NosTests/AuthorTests.swift

```diff
@@ -47,4 +47,101 @@ final class AuthorTests: CoreDataTestCase {
         author.nip05 = nip05WithUnderscore
         XCTAssertEqual(author.humanFriendlyIdentifier, "nip05.com")
     }
+
+    func test_hasNIP05_false_when_nip05_is_nil() throws {
```

Comment:
Our convention has been to make test names CamelCase without underscores, but I'm not against shifting the convention to this as its more readable, but it's something we should discuss. Thoughts @martindsq?

Reply:
I'm 100% for underscores since I like that they're more readable and really like that the first part is `test_methodName` where `methodName` is cased exactly the way the method is.